dig/README.md

```diff
@@ -0,0 +1,78 @@
+# Dependency Injection Graph
+
+This package provides a fairly opinionated way of resolving object dependencies.
```

Comment:
Can you start this with "Package dig provides" just to make it super clear that "Dependency Injection Graph" and "this package" and "dig" all mean the same thing?